core/transact.go

```diff
@@ -80,7 +81,13 @@ func (a *API) buildSingle(ctx context.Context, req *buildRequest) (*txbuilder.Te
 	maxTime := time.Now().Add(ttl)
 	tpl, err := txbuilder.Build(ctx, req.Tx, actions, maxTime)
 	if errors.Root(err) == txbuilder.ErrAction {
-		err = errors.WithData(err, "actions", errInfoBodyList(errors.Data(err)["actions"].([]error)))
+		// Format each of the inner errors contained in the data.
+		var formattedErrs []httperror.Response
+		for _, innerErr := range errors.Data(err)["actions"].([]error) {
```

Comment:
Nope. `txbuilder.ErrAction` is only returned if there are 1+ action-specific errors.